<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_AutomaticStudy_Default</class>
 <widget class="QMainWindow" name="Qt_AutomaticStudy_Default">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>样本学习软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>类名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ClassName"/>
          </item>
          <item>
           <widget class="QLabel" name="label_ClassType_Index">
            <property name="text">
             <string>分类索引</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ClassTyep_Index"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_PathSelect">
            <property name="text">
             <string>路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Path"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView_QueXianReady_List"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_insert">
            <property name="text">
             <string>插入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delete">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_rightmove">
            <property name="text">
             <string>右移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_leftmove">
            <property name="text">
             <string>左移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_study">
            <property name="text">
             <string>开始学习样本</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_MethodSelect">
            <item>
             <property name="text">
              <string>SVM</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>NN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_SaveResult">
            <property name="text">
             <string>学习结果保存</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_QueXianForStudy_List"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>预处理</string>
    </property>
    <addaction name="action_FilterSetting"/>
    <addaction name="action_MorphSetting"/>
    <addaction name="action_ThreshSetting"/>
    <addaction name="action_Preprocessing_Combination"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>特征选择</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Features_AutomaticSelect"/>
    <addaction name="action_Features_ManSelect_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>学习模型</string>
    </property>
    <addaction name="action_NewModel"/>
    <addaction name="action_OpenModel"/>
    <addaction name="action_SaveModel"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_FilterSetting">
   <property name="text">
    <string>滤波</string>
   </property>
  </action>
  <action name="action_MorphSetting">
   <property name="text">
    <string>形态变换</string>
   </property>
  </action>
  <action name="action_ThreshSetting">
   <property name="text">
    <string>阈值设定</string>
   </property>
  </action>
  <action name="action_Preprocessing_Combination">
   <property name="text">
    <string>组合设定</string>
   </property>
  </action>
  <action name="action_Features_AutomaticSelect">
   <property name="text">
    <string>自动选择</string>
   </property>
  </action>
  <action name="action_Features_ManSelect_2">
   <property name="text">
    <string>手动选择</string>
   </property>
  </action>
  <action name="action_NewModel">
   <property name="text">
    <string>新建模型</string>
   </property>
  </action>
  <action name="action_OpenModel">
   <property name="text">
    <string>打开学习模型</string>
   </property>
  </action>
  <action name="action_SaveModel">
   <property name="text">
    <string>保存学习模型</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>版本</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
